### PR TITLE
Uf 7271 refactor status and stakeholders

### PR DESCRIPTION
**Pull request by mha109**
Refactored all enums in the api to strings.

### CODE REVIEW

File: src/main/java/se/sundsvall/contract/api/model/enums/StakeholderRole.java [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-	public String getText() {
-		return text;
-	}
+	BUYER,
+	CONTACT_PERSON,
+	GRANTOR,
+	LAND_OWNER,
+	LEASE_HOLDER,
+	POWER_OF_ATTOURNEY_CHECK,
+	POWER_OF_ATTOURNEY_ROLE,
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/contract/api/model/enums/StakeholderRole.java:10]: Think it is spelled "Attorney".

File: pom.xml [hunk 2]
```diff
@@ -77,6 +78,14 @@
 			<artifactId>geojson-jackson</artifactId>
 			<version>${geojson-jackson.version}</version>
 		</dependency>
+		<!-- Needed until dept44 updates to at least 3.25 of this dependency -->
```
Comment (cheezi0747) [inline pom.xml:81]: why?

File: src/main/java/se/sundsvall/contract/api/model/enums/StakeholderRole.java (hunk 1 above)
Comment (mha109) [reply src/main/java/se/sundsvall/contract/api/model/enums/StakeholderRole.java:10]: oops, you are correct!

File: pom.xml (hunk 2 above)
Comment (mha109) [reply pom.xml:81]: To be able to use "withEnumStringComparison()" in tests. I'll put that in the comment as well

**Review by cheezi0747 — APPROVED** [review]

**Pull request merged**